presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeoFunctions.java

```diff
@@ -521,6 +522,25 @@ public static Slice stGeometryN(@SqlType(GEOMETRY_TYPE_NAME) Slice input, @SqlTy
         return serialize(ogcGeometry);
     }
 
+    @SqlNullable
+    @Description("Returns the Point geometry element at the specified index (indices started with 1)")
```

Comment:
`the Point geometry element` sounds weird; how about "Returns a vertex of the linestring at the specified index"?